tests/test_generic_frontend.py

```diff
@@ -0,0 +1,165 @@
+                    FrontendLayerType.Conv2d,
+                    FrontendLayerType.Activation,
+                ],
+                conv_kernel_sizes=[3, 5],
+                conv_strides=[1, 1],
+                conv_paddings=None,
+                conv_out_dims=[32, 32],
+                pool_kernel_sizes=[3],
+                pool_strides=[2],
+                pool_paddings=None,
+                activations=[nn.SiLU()],
+                out_features=384,
+                output_shape=[10, 25, 384],
+                in_sequence_mask=torch.Tensor(10 * [50 * [True] + 0 * [False]]).bool(),
+                out_sequence_mask=torch.Tensor(10 * [25 * [True] + 0 * [False]]).bool(),
+            ),
+            GenericFrontendV1TestParams(
+                batch=10,
+                time=50,
+                in_features=50,
+                layer_ordering=[
+                    FrontendLayerType.Conv2d,
+                    FrontendLayerType.Pool2d,
+                    FrontendLayerType.Activation,
+                    FrontendLayerType.Conv2d,
+                    FrontendLayerType.Activation,
+                ],
+                conv_kernel_sizes=[3, (3, 5)],
+                conv_strides=[1, (2, 1)],
+                conv_paddings=None,
+                conv_out_dims=[32, 32],
+                pool_kernel_sizes=[3],
+                pool_strides=[2],
+                pool_paddings=None,
+                activations=[nn.SiLU(), nn.SiLU()],
+                out_features=384,
+                output_shape=[10, 13, 384],
+                in_sequence_mask=torch.Tensor(10 * [50 * [True] + 0 * [False]]).bool(),
+                out_sequence_mask=torch.Tensor(10 * [13 * [True] + 0 * [False]]).bool(),
+            ),
+            GenericFrontendV1TestParams(
+                batch=10,
+                time=50,
+                in_features=50,
+                layer_ordering=[
+                    FrontendLayerType.Conv2d,
+                    FrontendLayerType.Pool2d,
+                    FrontendLayerType.Activation,
+                    FrontendLayerType.Conv2d,
+                    FrontendLayerType.Pool2d,
+                    FrontendLayerType.Activation,
+                ],
+                conv_kernel_sizes=[3, (3, 5)],
+                conv_strides=[1, (2, 2)],
+                conv_paddings=None,
+                conv_out_dims=[32, 32],
+                pool_kernel_sizes=[3, 5],
+                pool_strides=[(1, 2), (2, 3)],
+                pool_paddings=None,
+                activations=[nn.SiLU(), nn.SiLU()],
+                out_features=384,
+                output_shape=[10, 13, 384],
+                in_sequence_mask=torch.Tensor(10 * [50 * [True] + 0 * [False]]).bool(),
+                out_sequence_mask=torch.Tensor(10 * [13 * [True] + 0 * [False]]).bool(),
+            ),
+        ]
+    ):
+        print(idx)
+        shape, seq_mask = get_output_shape(test_params)
+        print(shape)
+        print(test_params.output_shape)
```

Comment:
```suggestion
        shape, seq_mask = get_output_shape(test_params)
```
Removing print statements.